pyband/messages/band/tss/v1/msgs.py

```diff
@@ -0,0 +1,91 @@
+from pyband.messages.base import BaseMessageWrapper
+from pyband.proto.band.tss.v1beta1 import (
```

Comment:
folder name for all modules should be v1beta1 (except oracle)? to align with proto.